apps/gauzy/src/app/pages/employee-levels/employee-level.component.html

```
@@ -119,19 +119,33 @@ <h5 class="title mr-3 ml-3">{{ 'POP_UPS.ADD' | translate }}</h5>
-				</button>
-				<button
-					(click)="save(addInput.value); disabled = true; ref.close()"
-					nbButton
-					status="success"
-					[disabled]="addInput.invalid"
-				>
-					{{ 'BUTTONS.SAVE' | translate }}
-				</button>
+				<div class="col-sm-12 d-flex">
+					<button
+						class="delete mr-3 ml-3"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)